Source/WebKit/DerivedSources.make

```diff
@@ -334,6 +334,7 @@ EXTERNAL_FLAGS := -DRELEASE_WITHOUT_OPTIMIZATIONS $(addprefix -D, $(GCC_PREPROCE
 platform_h_compiler_command = $(CC) -std=c++2a -x c++ $(1) $(SDK_FLAGS) $(TARGET_TRIPLE_FLAGS) $(FRAMEWORK_FLAGS) $(HEADER_FLAGS) $(EXTERNAL_FLAGS) -include "wtf/Platform.h" /dev/null
 
 FEATURE_AND_PLATFORM_DEFINES := $(shell $(call platform_h_compiler_command,-E -P -dM) | $(PERL) -ne "print if s/\#define ((HAVE_|USE_|ENABLE_|WTF_PLATFORM_)\w+) 1/\1/")
+FEATURE_AND_PLATFORM_UNDEFINES := $(shell $(call platform_h_compiler_command,-E -P -dM) | $(PERL) -ne "print if s/\#define ((HAVE_|USE_|ENABLE_|WTF_PLATFORM_)\w+) 0/\1/")
```

Comment:
This should do it in one compiler invocation:

```make
FEATURE_AND_PLATFORM_FLAGS := $(shell $(call platform_h_compiler_command,-E -P -dM) | $(PERL) -ne "print if s/\#define ((HAVE_|USE_|ENABLE_|WTF_PLATFORM_)\w+) (\d)/\1=\2/")
FEATURE_AND_PLATFORM_DEFINES := $(patsubst %=1, %, $(FEATURE_AND_PLATFORM_FLAGS))
FEATURE_AND_PLATFORM_UNDEFINES := $(patsubst %=0, %, $(FEATURE_AND_PLATFORM_FLAGS))
```

Reply:
Aha, thanks. My make skills are... rudimentary 🙄

Reply:
Consider yourself lucky!